### PR TITLE
Add ability to collapse directories

### PR DESCRIPTION
**Pull request by bign8**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions

### Description of the Change

Add the ability to collapse directories.

collapsible-directories

### Alternate Designs

No alternative designs were considered, but I'm open to suggestions!

### Benefits

Users could collapse large and/or complicated directory structures.

### Possible Drawbacks

None that I am aware of.

### Applicable Issues

#55 - This isn't full tree-view integration, but it may be enough to feel like it.

### CODE REVIEW

File: lib/directory-view.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
 /** @babel */
```
Comment (Koenie1980) [inline lib/directory-view.js]: bign8:collapsible-directories

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Comment by EllaMartinez1** [comment]
Ella-Intelligent Global Network Inc.
[Ella-Intelligent.org.txt](https://github.com/atom/archive-view/files/4197265/Ella-Intelligent.org.txt)

**Comment by danielleoking7474** [comment]
#[bign8:collapsible-directories]()

**Comment by danielleoking7474** [comment]
`

`

**Comment by ThatXliner** [comment]
Please merge!

**Review by ghost — APPROVED** [review]

**Comment by bign8** [comment]
I'm not quite sure what the Atom maintainers are looking for, so I tried to reach out on [the Atom discussions page](https://github.com/atom/atom/discussions/22119), maybe that can stir some interest.

**Review by sadick254 — APPROVED** [review]
This looks good. Thank you for adding tests around this.

**Comment by antonyggvzvmnxxcx** [comment]
> ### Requirements
> * Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
> * All new code requires tests to ensure against regressions
> 
> ### Description of the Change
> Add the ability to collapse directories.
> 
> collapsible-directories
> 
> ### Alternate Designs
> No alternative designs were considered, but I'm open to suggestions!
> 
> ### Benefits
> Users could collapse large and/or complicated directory structures.
> 
> ### Possible Drawbacks
> None that I am aware of.
> 
> ### Applicable Issues
> #55 - This isn't full tree-view integration, but it may be enough to feel like it.

**Comment by karatalaya** [comment]
https://github.com/symphonists/html5_doctype

**Pull request merged**